src/main/scala/beam/utils/scripts/FailFast.scala

```diff
@@ -8,19 +8,21 @@ object FailFast extends LazyLogging {
 
   def run(beamServices: BeamServices): Unit = {
 
+    val c = beamServices.beamConfig
```

Comment:
can we rename this to config instead of "c"?

Reply:
ok